lib/ecto_job/job_queue.ex

```diff
@@ -54,6 +54,7 @@ defmodule EctoJob.JobQueue do
           max_attempts: integer | nil,
           params: map(),
           notify: String.t() | nil,
+          priority: integer | nil,
```

Comment:
Since `priority` has a default value of `0`, we can leave of the `| nil` here.

Reply:
@mbuhot Done!